operator/EXTENDING_CRDS.md

```diff
@@ -122,3 +122,19 @@ Some data types are discouraged, in particular:
 * Nullability: See [Kubernetes Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting-and-nullable).
 * Late initialization: See [API Conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#late-initialization).
 * Labels, Selector and Annotations: See [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions).
+
+## Crd-schema-checker
```

Comment:
How about adding this paragraph right after the `Breaking changes` one, rather than hidden away below style recommendations?

Reply:
Done